### PR TITLE
Bump to v0.1.0

### LINKED ISSUE #4: Add SideDrawer Component
Create a new SideDrawer component

- Animation transition time should be configurable

### LINKED ISSUE #5: Add a ref props to all input elements
Since components have a ref on their own, this prop should be called elementNameRef for all form element components

### LINKED ISSUE #7: Switch component does not properly set --disabled className when isDisabled is true
Note that gooey-checkbox--disabled is set, but not gooey-switch--disabled. Example:

`<div class="gooey-checkbox gooey-checkbox--disabled gooey-switch">`

### PR DESCRIPTION
**Pull request by mmellado**
- Adding SideDrawer component (closes #4 )
- Added optional ref prop to all form elements (closes #5 )
- Adding disabled and on classNames to Switch (closes #7)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
 
-[Diff](https://github.com/airtame/airtame-gooey-react/compare/v0.0.0...v0.0.1)
+[Diff](https://github.com/airtame/airtame-gooey-react/compare/v0.0.2...v0.1.0)
 
-- Fixed the RadioGroup component (Now current value changes properly)
-- Updated Button component to extend React.Component. This allows unsing the ref prop
-- Added onChange callback to the TextArea component
-- Added onChange callback to the TextField component
+- Added SideDrawer component
+- Added ref prop to all form elements
```
Comment (elvisvoer) [inline CHANGELOG.md]: You forgot to add the "Adding disabled and on classNames to Switch" commit to the changelog.

File: Checkbox/Checkbox.js [hunk 2]
```diff
@@ -52,6 +52,14 @@ export default class Checkbox extends Component {
      */
     onChange: PropTypes.func,
     /**
+     * ref for the input element
+     * @type {function | string}
+     */
+    inputRef: PropTypes.oneOfType([
+      PropTypes.func,
+      PropTypes.string,
```
Comment (elvisvoer) [inline Checkbox/Checkbox.js]: Considering that it is a bad practice to use refs as strings and they will be later deprecated should we allow our input refs to be strings or just enforce it as a function?

Comment (mmellado) [inline Checkbox/Checkbox.js]: This is mainly for testing purpose based on how enzyme recommends testing refs. Let's leave as is for now and decide if we should update once React deprecates string refs

File: CHANGELOG.md (hunk 1 above)
Comment (mmellado) [inline CHANGELOG.md]: Adding :)

**Pull request merged**